### PR TITLE
fix(LookMovie): presence not working

### LINKED ISSUE #4974: LookMovie | lookmovie.io
### Description

When I visit LookMovie.io with the presence enabled it shows in discord that I am browsing the home page but as soon as I start watching something the presence will disappear.
I think the presence disappears because for some reason now when you start watching you are being redirected to a different domain.

### Steps to Reproduce

1. Visit LookMovie.io with the presence enabled
2. Start watching the home page you will notice the presence appears.
3. Start watching any move or any series and you will notice the presence disappears.

### Expected Behavior

The expected behaviour is that it will show the presence will appear while watching and it will display what movie or series your watching and at what time in the movie/serie you are and of course if you paused it yes or no.

### Screenshots

Screenshot_294

### Operating System (OS)

Windows 11

### Browser

Chrome, Opera GX

### Browser Version

Google Chrome: 96.0.4664.45 / Opera GX: LVL3 (core: 81.0.4196.61)

### PR DESCRIPTION
**Pull request by DarkVillager**
Resolves #4974
**Describe the changes in this pull request:**
- New application, old one didnt work anymore.
- Fixed LookMovie's Browse
- Added new url (for watching)
- Fixed Timestamps
- Fixed Browsing String
- Took Over Presence:
image

<details>
<summary> Proof showing the creation/modification is working as expected </summary>
<br>

</details>

### CODE REVIEW

**Comment by EncryptedDev** [comment]
Presence contains pirated content which is illegal. 

Reference: [Article](https://techyjungle.com/lookmovie-ag-guide/#:~:text=By%20reading%20the%20description%2C%20we%20can%20conclude%20that%20any%20movie%20or%20TV%20show%20not%20streamed%20directly%20by%20the%20production%20house%20is%20illegal%20and%20cannot%20be%20legally%20accessed.%20In%20the%20same%20way%2C%20Lookmovie%20is%20an%20illicit%20piracy%20website) & [Quora](https://www.quora.com/Why-is-Lookmovie-ag-not-working)

**Comment by EncryptedDev** [comment]
Draft until discussed further

File: websites/L/LookMovie/dist/metadata.json [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
 	"$schema": "https://schemas.premid.app/metadata/1.6",
 	"author": {
-		"name": "PreMiD",
-		"id": "503557087041683458"
+		"name": "Dark_Ville",
+		"id": "638080361179512853"
 	},
+	"contributors": [
+		{
+			"name": "PreMiD",
```
Comment (EncryptedDev) [inline websites/L/LookMovie/dist/metadata.json:9]: Remove this

File: websites/L/LookMovie/presence.ts [hunk 2]
```diff
@@ -1,55 +1,65 @@
...
-				document.querySelector(".seasons-switcher > span").textContent
-			} ${document.querySelector(".episodes-switcher > span").textContent}`;
-		} else if (document.location.pathname.includes("/movies/view")) {
-			presenceData.details = document.querySelector(
-				".watch-heading > h1 > span"
-			).previousSibling.textContent;
-			presenceData.state = document.querySelector(
-				".watch-heading > h1 > span"
-			).textContent;
+		video: HTMLVideoElement = document.querySelector("#video_player"),
```
Comment (EncryptedDev) [inline websites/L/LookMovie/presence.ts:14]:
```suggestion
		video = document.querySelector<HTMLVideoElement>("#video_player"),
```

File: websites/L/LookMovie/presence.ts [hunk 3]
```diff
@@ -1,55 +1,65 @@
...
+		videoDur = document.querySelector(
+			"#video_player > div.vjs-control-bar > div.vjs-duration.vjs-time-control.vjs-control > span.vjs-duration-display"
+		);
+	if (video !== null && videoDur !== null) {
```
Comment (EncryptedDev) [inline websites/L/LookMovie/presence.ts:18]:
```suggestion
	if (video && videoDur) {
```

File: websites/L/LookMovie/presence.ts [hunk 4]
```diff
@@ -1,55 +1,65 @@
...
+		const titles = document.querySelector(
```
Comment (EncryptedDev) [inline websites/L/LookMovie/presence.ts:19]: Use generic types here too

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/L/LookMovie/presence.ts [hunk 5]
```diff
@@ -1,55 +1,65 @@
...
-				".watch-heading > h1 > span"
-			).textContent;
+		video = document.querySelector<HTMLVideoElement>("#video_player"),
+		videoDur = document.querySelector(
+			"#video_player > div.vjs-control-bar > div.vjs-duration.vjs-time-control.vjs-control > span.vjs-duration-display"
+		);
+	if (video && videoDur) {
+		const titles = document.querySelector(
+				'meta[property="og:title"]'
+			) as HTMLMetaElement,
```
Comment (Slowlife01) [inline websites/L/LookMovie/presence.ts:21]:
```suggestion
		const titles = document.querySelector<HTMLMetaElement>(
				'meta[property="og:title"]'
			),
```

File: websites/L/LookMovie/presence.ts [hunk 6]
```diff
@@ -1,55 +1,65 @@
 const presence = new Presence({
-		clientId: "760588725494218844"
+		clientId: "934789855962083359"
 	}),
 	strings = presence.getStrings({
 		play: "presence.playback.playing",
 		pause: "presence.playback.paused",
```
Comment (Bas950) [inline websites/L/LookMovie/presence.ts:6]: use general strings

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Comment by evilstriker** [comment]
The function of displaying movies and series in the Discord presence is not functional.

**Comment by DarkVillager** [comment]
> The function of displaying movies and series in the Discord presence is not functional.

will check out. Seems like only the url changed

**Pull request merged**